util/gcs/sort.go

```diff
@@ -93,3 +94,19 @@ func Touch(ctx context.Context, client ConditionalClient, path Path, generation
 	cond.DoesNotExist = true
 	return client.If(&cond, &cond).Upload(ctx, path, genZero, DefaultACL, "no-cache")
 }
+
+// Sort the builds by monotonically decreasing original prefix base name.
+//
+// In other words,
```

Comment:
Please add this as a test case also. The current coverage doesn't make it clear how, for example, b/10 and c/1 should interact.

Reply:
Please read the other tests in this package, which already cover this scenario.

Ensuring that the `Build` for `gs://c/10` sets the `baseName` to 10 is not the responsibility of `Sort()`